roles/cnf_cert/defaults/main.yml

```diff
@@ -23,7 +23,7 @@ tnf_image_suffix: "{{ lookup('community.general.random_string', length=16, speci
 # Provide feedback for each tnf test launched. Just copy this template in your pipeline file and fill the value for the
 # tests you want to provide feedback. You don't need to include the tests that you don't want to provide feedback.
 # To insert line breaks, use \\n (you need to escape it). Same for quotes and so on.
-# List of tests updated until tnf v5.0.5
+# List of tests updated until tnf v5.0.6
 tnf_feedback:
```

Comment:
tnf_feedback must be updated for this version to include some new tests added for tnf v5.0.6:
```
operator-crd-openapi-schema
operator-semantic-versioning
```

Reply:
Just a heads up, have some additional `operator-` tests in flight and we weren't going to bump the minor version to v5.1.x until they were all available for use.

Reply:
> Thanks for the heads up, I see the tests are passing in both TNF jobs, so I was wondering what could be a relevant feedback to add? Perhaps something about possible failure and introduction in 5.0.6?

It's just to add an entry in the tnf_feedback default value, which is an empty string for all tests available. This is mainly to have a kind of template that can be used afterwards to fill in the tests that are really failing

Reply:
Thanks for the clarification, fixed in the last commit!